packages/mobile/src/home/WalletHome.tsx

```diff
@@ -90,12 +96,28 @@ export class WalletHome extends React.Component<Props> {
     this.props.refreshAllBalances()
   }
 
-  componentDidMount() {
+  componentDidMount = async () => {
```

Comment:
so this is a "fire and forget" situation where you want to trigger tryImportContacts but don't really care about the result. i'm assuming the linter complained about an unhandled promise if you just call `this.tryImportContacts()` - which you could add a catch to if you wanted to. just noting the semantics here since we don't really care about awaiting the result